tests/test_session.py

```diff
@@ -0,0 +1,15 @@
+from ReactiveXComponent.communication.xcSession import *
```

Comment:
like in the js api, separate unit tests from "testing code" in different sub-folders